contributing.md

```diff
@@ -0,0 +1,51 @@
+Contributing to the Certificate Linux System Role
+=============================================
+
+Where to start
+--------------
+
+The first place to go is [Contribute](https://linux-system-roles.github.io/contribute.html).
+This has all of the common information that all role developers need:
+* Role structure and layout
+* Development tools - How to run tests and checks
+* Ansible recommended practices
+* Basic git and github information
+* How to create git commits and submit pull requests
+
+- **Bugs and needed implementations** are listed on [Github
+  Issues](https://github.com/linux-system-roles/certificate/issues). Issues labeled with
+[**help
+wanted**](https://github.com/linux-system-roles/certificate/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+are likely to be suitable for new contributors!
+
+- **Code** is managed on
+  [Github](https://github.com/linux-system-roles/certificate), using [Pull
+Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
+
+- The code needs to be **compatible with the Python versions supported by the role platform**.
```

Comment:
@rjeffman check the new text - I think it better explains the python support - let me know.

Reply:
Text is much better now. Thank you for the change.

I hear, a lot, about the mess that Python versioning is right now, mostly due to legacy systems/older distros, and the current text make it a bit clearer.